pkg/cmd/roachprod/main.go

```diff
@@ -337,19 +337,16 @@ hosts file.
 				return clusterVMs[0].MachineType
 			}
 			cpuArch := func(clusterVMs vm.List) string {
-				// Display CPU architecture and family.
-				if clusterVMs[0].CPUArch == "" {
-					// N.B. Either a local cluster or unsupported cloud provider.
-					return ""
-				}
+				// Building CPU info column to displays "CPU family (CPU arch)"
```

Comment:
Damn, I'm 100% clueless about CPU family <-> arch mapping :D 
The reasoning doesn't work hold for indicating FIPS, though, since it would just state `ice lake`.

Adding 5 letters seemed reasonable, but I can revert.